org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/util/PluginProjectFactory.java

```diff
@@ -130,6 +131,9 @@ public String apply(String input) {
 
 	protected void createManifest(IProject project, IProgressMonitor progressMonitor) throws CoreException {
 		final StringBuilder content = new StringBuilder("Manifest-Version: 1.0\n");
+		if (JavaVersion.fromBree(getBreeToUse()).isAtLeast(JavaVersion.JAVA8)) {
```

Comment:
Why not add always?

Reply:
cause it might confuse users

Reply:
If we want to have a guard, it should be at least Java9 I suppose. But I’m with @kthoms here - guess it’s safe to have it for older versions, too. As long as it does not produce a warning.

Reply:
I removed the condition. It started with java 9 but the fact that it is ignored for java 8 have let me to the wrong conclusion that it was a java 8 thing.